templates/web/base/maps/_map_instructions.html

```diff
@@ -0,0 +1,18 @@
+<div class="keyboard-intructions-wrapper basic">
+    <div>
+        <div class="keyboard-key-icon zoom-in">Zoom in</div>
```

Comment:
On second thoughts, the text labels should also use the name of the _key_ rather than _what the key will do_, so:

```
<div class="keyboard-key-icon zoom-in">[% loc("plus") %]</div>
```